src/theme.tsx

```diff
@@ -149,6 +149,8 @@ export const condaStoreTheme = createTheme(baseTheme, {
   }
 });
 
+export const theme = condaStoreTheme;
```

Comment:
Is there any fundamental change/impact of one or the other approach aside from a double export?

Reply:
Heh, me asking the question was really a tell that I shouldn't alias the theme. I pushed a commit that fixes the tests without doing this. It wasn't hard.

Reply:
And the reason why I think I shouldn't alias it is because it could make it harder to trace where that particular theme is being used across the code base (much easier to search the string "condaStoreTheme" than "theme")